lib/instance-validator.js

```diff
@@ -321,10 +321,7 @@ class InstanceValidator {
     let error;
 
     if (rawAttribute.allowNull === false && (value === null || value === undefined)) {
-      const validators = this.modelInstance.validators[field];
-      const errMsg = validators
-                      ? (validators.notNull || {}).msg
-                      : `${field} cannot be null`;
+      const errMsg = `${field} cannot be null`;
```

Comment:
@felixfbecker in my project all entites haven't custom not null validation message.
So what the next, do you approve it?